### ISSUE TITLE
ConnectionRefusedError on sending 2nd datagram

### ISSUE THREAD

**Issue by pasdVn** [jsbronder/asyncio-dgram#13]
Hi,

I am quite new to asyncio an want to use asyncio-dram for a little project. However I am facing the following problem, being not sure, if I am missing something or if this is some issue within asyncio-dgram.

As example I modified the example UDP client a bit:
```import asyncio

import asyncio_dgram


async def udp_echo_client():
    stream = await asyncio_dgram.connect(("127.0.0.1", 8888))

    while True:
        await stream.send(b"Hello World!")
        print("sent...")
        await asyncio.sleep(1)

    stream.close()


def main():
    loop = asyncio.get_event_loop()
    loop.run_until_complete(udp_echo_client())


if __name__ == "__main__":
    main()
```

which results in

```(venv) tobias@hugo:~/Entwicklung/gopro-mavlink$ ./debug.py 
sent...
Traceback (most recent call last):
  File "./debug.py", line 26, in <module>
    main()
  File "./debug.py", line 22, in main
    loop.run_until_complete(udp_echo_client())
  File "/usr/lib/python3.8/asyncio/base_events.py", line 608, in run_until_complete
    return future.result()
  File "./debug.py", line 13, in udp_echo_client
    await stream.send(b"Hello World!")
  File "/home/tobias/Entwicklung/gopro-mavlink/venv/lib/python3.8/site-packages/asyncio_dgram/aio.py", line 152, in send
    await super()._send(data)
  File "/home/tobias/Entwicklung/gopro-mavlink/venv/lib/python3.8/site-packages/asyncio_dgram/aio.py", line 106, in _send
    _ = self.exception
  File "/home/tobias/Entwicklung/gopro-mavlink/venv/lib/python3.8/site-packages/asyncio_dgram/aio.py", line 59, in exception
    raise exc
  File "/usr/lib/python3.8/asyncio/selector_events.py", line 999, in _read_ready
    data, addr = self._sock.recvfrom(self.max_size)
ConnectionRefusedError: [Errno 111] Connection refused
```

i.e. this obviously happens, if I try to send a 2nd consecutive datagramm without `recv()` something before!?
Am I missing something?

**Comment by jsbronder** [comment]
You dropped the `udp_echo_server` which means that your client has nothing to connect to.  In particular, this line ` stream = await asyncio_dgram.connect(("127.0.0.1", 8888))`.

Given that UDP is connectionless, the `connect` call does not fail immediately, nor does the first `send`.  But, the remote side will send an ICMP packet signifying that no one is listening which will cause successive `send`/`recv` calls to fail.

tl;dr You need something listening for UDP packets at 127.0.0.1:8888.  That's what the `udp_echo_server` in the example does.

**Comment by pasdVn** [comment]
Ah ok - got that. Thanks for the clarification.
